server/functions/date.go

```diff
@@ -106,3 +108,42 @@ var date_cmp = framework.Function2{
 		return int32(ab.Compare(bb)), nil
 	},
 }
+
+// getDateStyleInputFormat returns set the defined format in DateStyle config as the first in the ordered list of date parsing modes.
```

Comment:
`...returns set`
I think this is a typo? I'm assuming you just meant `sets` here